### ISSUE TITLE
pod install error: The Swift pod `react-native-leanplum` depends upon `Leanplum-iOS-SDK`, which does not define modules.

### ISSUE THREAD

**Issue by defualt** [Leanplum/Leanplum-ReactNative-SDK#4]
"react-native": "0.62.2"
"@leanplum/react-native-sdk": "1.0.10",

```
npm install @leanplum/react-native-sdk --save
(cd ios && pod deintegrate && pod install)
```

```
[!] The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `react-native-leanplum` depends upon `Leanplum-iOS-SDK`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
```

To fix this I added this to my project's Podfile
```
pod 'Leanplum-iOS-SDK', :modular_headers => true
```

**Comment by barsumek** [comment]
I've also encountered this problem, would be nice to add info about it to README or fix it in the SDK.

Worth to note -> setting `use_modular_headers!` is not a good idea, because, at least for me, it causes a lot of issues for React Native dependencies such as `glog` or `Folly` and forces to manually set `:modular_headers => false` for them. Setting `true` only for `Leanplum-iOS-SDK` pod is a simpler solution.

**Comment by e7mac** [comment]
My understanding is that this commit https://github.com/Leanplum/Leanplum-iOS-SDK/commit/7ddb9f03aefe80af0e9118b635a8a407ca4467f7 should fix it. Please let me know if that is incorrect. We are QAing a release with this fix.

**Comment by e7mac** [comment]
Fixed in [v1.0.11](https://github.com/Leanplum/Leanplum-ReactNative-SDK/releases/tag/1.0.11)
